packages/core/ledger/snapshot.go

```diff
@@ -8,9 +8,9 @@ import (
 
 // Snapshot represents a snapshot of the current ledger state.
 type Snapshot struct {
-	Header              *SnapshotHeader            `serix:"0"`
-	OutputsWithMetadata []*OutputWithMetadata      `serix:"1,lengthPrefixType=uint32"`
-	EpochDiffs          map[epoch.Index]*EpochDiff `serix:"2,lengthPrefixType=uint32"`
+	Header         *SnapshotHeader            `serix:"0"`
+	FullUTXOStates []*OutputWithMetadata      `serix:"1,lengthPrefixType=uint32"`
```

Comment:
I think the old name was more descriptive of the objects actually contained in the field!